### PR TITLE
HR1 module update

### PR DESCRIPTION
**Pull request by ajtheb**
Appraisal models added

### CODE REVIEW

File: FusionIIIT/applications/establishment/models.py [hunk 1]
```diff
@@ -167,4 +183,98 @@ def elsewhere_ltc_remaining(self):
...
+    publications = models.CharField(max_length=200, blank=True, null=True, default='')
+    conferences_meeting_attended = models.CharField(max_length=200, blank=True, null=True, default='')
+    conferences_meeting_organized = models.CharField(max_length=200, blank=True, null=True, default='')
+    admin_assign=models.CharField(max_length=20, blank=True, null=True, default='')
+    sevice_to_ins=models.CharField(max_length=20, blank=True, null=True, default='')
+    extra_info = models.CharField(max_length=200, blank=True, null=True, default='')
+    faculty_comments= models.CharField(max_length=200, blank=True, null=True, default='')
+    
+
+class courses_instructed(models.Model):
```
Comment (adi0311) [inline FusionIIIT/applications/establishment/models.py]: Class name should be in Pascal Case.

File: FusionIIIT/applications/establishment/models.py [hunk 2]
```diff
@@ -167,4 +183,98 @@ def elsewhere_ltc_remaining(self):
 
     def __str__(self):
         return str(self.user.username) + ' - joined on ' + str(self.date_of_joining)
+
+class Appraisal(models.Model):
```
Comment (adi0311) [inline FusionIIIT/applications/establishment/models.py:187]: Add docstring to every model's class.

File: FusionIIIT/applications/establishment/models.py [hunk 3]
```diff
@@ -167,4 +183,98 @@ def elsewhere_ltc_remaining(self):
...
+    reg_students=models.IntegerField(blank=True, null=True)
+    co_instructor=models.ForeignKey(User, related_name='co_inst',
+                                    on_delete=models.CASCADE)
+
+class NewCoursesOffered(models.Model):
+    appraisal = models.ForeignKey(Appraisal, related_name='applicants_offered_new_courses',
+                                  on_delete=models.CASCADE)
+    course_name = models.CharField(max_length=30)
+    course_num=models.IntegerField(blank=True, null=True)
+    UGorPG=models.CharField(max_length=2,blank=True, null=True)
```
Comment (adi0311) [inline FusionIIIT/applications/establishment/models.py]: Fields name should be in `snake_case`.

**Review by adi0311 — CHANGES_REQUESTED** [review]

File: FusionIIIT/applications/establishment/models.py (hunk 1 above)
Comment (ajtheb) [reply FusionIIIT/applications/establishment/models.py]: Please check now.

File: FusionIIIT/applications/establishment/models.py (hunk 3 above)
Comment (ajtheb) [reply FusionIIIT/applications/establishment/models.py]: please check now

**Pull request merged**